src/registry/radio/radio-validation.stories.ts

```diff
@@ -52,8 +52,42 @@ export const ManualMinimumOneValue: Story = {
       await userEvent.type(labelInput, 'Foo');
       await userEvent.clear(labelInput);
       await userEvent.keyboard('[Tab]');
-      await expect(await canvas.findByText('The option label is a required field.')).toBeVisible();
-      await expect(await canvas.findByText('The option value is a required field.')).toBeVisible();
+
+      await waitFor(async () => {
+        await expect(
+          await canvas.findByText('The option label is a required field.')
+        ).toBeVisible();
+        await expect(
+          await canvas.findByText('The option value is a required field.')
+        ).toBeVisible();
+      });
+    });
```

Comment:
these often fail locally in storybook when you don't have focus on the browser window. could you check if they're really necessary? I can't really remember these tests failing in CI, while the file size validation tests are notable sources of flakiness

Reply:
Its a bit weird/scary that some tests (only) fail locally.. I'll check them out and remove the `await`'s and `waitFor`'s that aren't necessary